src/test/http_server_spec/README.md

```diff
@@ -0,0 +1,15 @@
+# HTTP server Spec
+
+A minimal suite of acceptance tests for the HTTP server learning project.
+
+### Installation
+
+Install [Bundler](https://bundler.io/), and then run `bundle install` to clone dependencies.
```

Comment:
what ruby version?

Reply:
There is a `.ruby-version` file included in the repository: https://github.com/8thlight/http_server_spec/blob/master/.ruby-version

Should I add the version in the `README`?

Reply:
It's good practice to do so. That ruby version does it for rbenv folk, but if they don't they need to know.